components/forms/CreateTxForm/MsgForm/MsgInstantiateContractForm.tsx

```diff
@@ -102,6 +107,13 @@ const MsgInstantiateContractForm = ({
         return false;
       }
 
+      try {
+        macroCoinToMicroCoin({ denom, amount }, chain.assets);
```

Comment:
I'm confused by the name of this function because the case in https://github.com/cosmos/cosmos-multisig-ui/issues/183 shows we have a base denom in [atto](https://en.wikipedia.org/wiki/Metric_prefix#List_of_SI_prefixes). 

Should this be renamed to `displayCoinToBaseCoin` or similar?

Reply:
Yeah the implementation has exceeded the logic behind the initial name I gave it